### PR TITLE
Improve a11y compliance for podcast external links

### PR DESCRIPTION
**Pull request by pharingee**
Resolves #NUMBER

**Overall change:**
Make podcast external links a11y compliant

**Code changes:**
- Replace all 1px values with 0.0625rem.
- Move component to a labelled aside landmark (not within main landmark).
- Use an unordered list for the links (if more than 1 link).
- Announce brand title as part of the link text (with VH comma).
- Check hover and focus styling works
- Update h2 to use Section label

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [x] This PR requires manual testing

### CODE REVIEW

**Comment by greenc05** [comment]
Please could we fix up the snapshot test @pharingee doesnt' look like the right component here?

File: src/app/containers/PodcastExternalLinks/index.jsx [hunk 1]
```diff
@@ -33,30 +64,53 @@ const PodcastExternalLinks = ({ links }) => {
     ['media', 'podcastExternalLinks'],
     translations,
   );
+  const moreThanOneLink = links.length > 1;
```
Comment (jroebu14) [inline src/app/containers/PodcastExternalLinks/index.jsx]: `hasMultipleLinks` would sounds better

**Review by jroebu14 — APPROVED** [review]

**Comment by greenc05** [comment]
Did you check it looks ok with no CSS @pharingee?

**Comment by greenc05** [comment]
Out of interest why is this component not in psammead?

**Comment by pharingee** [comment]
Screen Shot 2021-03-08 at 3 58 21 PM

This is how it looks with CSS disabled.

> Out of interest why is this component not in psammead?

We are currently working on an experiment to keep non-reusable components out of psammead. Something like this which will only ever be used by simorgh should be kept in simorgh with reusable components in psammead. This will make it easier to debug and develop. If this issue were to happen in psammead we would have had to publish a new version in psammead and then update the version in simorgh. The latter part of this issue explains it better: https://github.com/bbc/psammead/issues/4338

**Comment by greenc05** [comment]
Thank you @pharingee Bring on simorgh preview urls.

**Review by RichardPK — COMMENTED** [review]
I'm not sure the styling above 600px is right here G:

The `StyledSectionLabel` looks like it should have a 1rem top-margin, it currently has 1.5rem
Screenshot 2021-03-08 at 17 11 12

There should also be a 1rem bottom margin between the links and the bottom-border, I think the span is giving it `0.8125rem`
Screenshot 2021-03-08 at 17 12 14

The spacing between the external links component and the recent episodes below also looks out. On the designs it looks like it's after a 2.5rem margin at 400px, and 2rem at 600px. The PR only has the recent episodes' SectionLabel top margin, which is 1.5rem.
Screenshot 2021-03-08 at 17 15 24

**Comment by pharingee** [comment]
> There should also be a 1rem bottom margin between the links and the bottom-border, I think the span is giving it 0.8125rem

The line spacing adds the rest of the padding to make it 1rem.

**Comment by RichardPK** [comment]
> The line spacing adds the rest of the padding to make it 1rem.

Not sure it does man, the distance between the `sectionlabel` and the top border, and the `links` and the bottom border, should be the same (design):

Screenshot 2021-03-09 at 09 30 30

But visually it's quite clear it isn't. I don't think the line height adds much at all to the span margin, but it does add a bit for the sectionlabel:

Screenshot 2021-03-09 at 09 28 52

**Comment by pharingee** [comment]
@RichardPK  Take a look at https://zpl.io/2Enlwqn it should be clearer.

Screen Shot 2021-03-09 at 9 49 59 AM

**Review by RichardPK — APPROVED** [review]
As discussed - let's merge the changes and see what UX says about the spacing 👌

**Pull request merged**